### PR TITLE
Resolve UTF8 problem with PowerShell output

### PR DESCRIPTION
**Pull request by ahotton**
Can you push the resolution of the UTF8 problem on the master branch please.

### CODE REVIEW

**Comment by TraGicCode** [comment]
Can you please provide more information on what system you are having the issue on?

File: lib/puppet/provider/windowsfeature/default.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def self.instances
     result = if win2008 == true
                ps('Import-Module ServerManager; Get-WindowsFeature | Select Name,Installed | ConvertTo-XML -As String -Depth 4 -NoTypeInformation')
              else
-               ps('Import-Module ServerManager; Get-WindowsFeature | ConvertTo-XML -As String -Depth 4 -NoTypeInformation')
+               ps('Get-WindowsFeature | Select Name,Installed | ConvertTo-XML -As String -Depth 4 -NoTypeInformation')
```
[the remark this answers is not in the document]
Comment (BertrandJO) [reply lib/puppet/provider/windowsfeature/default.rb]: It works on a Windows Server 2012 R2 FR 

before :

...
Exception parsing
Line: 5
Position: 154
Last 80 unconsumed characters:
Acc?s ? distance</Property>
Line: 5
Position: 154
Last 80 unconsumed characters:
Acc?s ? distance</Property>

after :

......
Notice: /File[C:/ProgramData/PuppetLabs/puppet/var/lib/puppet/provider/windowsfeature/default.rb]/content: content changed '{md5}136f6129659cdbda75b6e73f41c39379' to '{md5}45c201f4597da588e2e27f63cf72ab6d'


......
Notice: Finished catalog run in 3.08 seconds

[the remark this answers is not in the document]
Comment (BertrandJO) [reply lib/puppet/provider/windowsfeature/default.rb]: and the output for the Get-WindowsFeature gets down from 1,5 Mo to few Ko, great !

[the remark this answers is not in the document]
Comment (ahotton) [reply lib/puppet/provider/windowsfeature/default.rb]: @BertrandJO has said everything.

Without this modification, you can not run the module on WINSRV 2K12 FR.

[the remark this answers is not in the document]
Comment (TraGicCode) [reply lib/puppet/provider/windowsfeature/default.rb]: Thanks for the info.  Give me a little bit of time as i try and juggle this with work + freetime + family!  I need to rebuild my packer image for beaker acceptance testing so i can verify this!

[the remark this answers is not in the document]
Comment (ahotton) [reply lib/puppet/provider/windowsfeature/default.rb]: Of course, no problem.
Thanks.

[the remark this answers is not in the document]
Comment (TraGicCode) [reply lib/puppet/provider/windowsfeature/default.rb]: I have created my vagrant box using packer with the french version.  I will let you know what i find.  i will not be able to look at it today though :(

**Comment by TraGicCode** [comment]
@ahotton Can you please allow me to make changes to your branch in your fork?  I've tested your changes and refactored it a bit.

**Comment by TraGicCode** [comment]
I have refactored your changes and everything seems to be working now in regards to the UTF8 error you were experiencing.  @bastelfreak Are you able to review this?

**Comment by ahotton** [comment]
Thanks @TraGicCode for your work.

**Pull request merged**